### ISSUE TITLE
need `DataFrame.dropna` function to remove missing rows

### ISSUE THREAD

**Issue by episodeyang** [Gmousse/dataframe-js#74]
Would be great if there is a performant way to do this.

The documentation on the API is comprehensive. There exists `drop` and `dropDuplicates`, so `dropna` would probably be a good addition.

Right now I'm achieving this functionality with [filter](https://gmousse.gitbooks.io/dataframe-js/doc/api/dataframe.html#filter):

```javascript
const df = new DataFrame({
    y: series.yMean,
    x: series.xData ? series.xData : series.yMean.map((_, i) => i)
  });
  return df
      .filter(row => row.get('y') === row.get('y'))
      .toCollection();
```

Given that this is a pure javascript library, the performance of this is probably the best you can get. On the other hand, it might get more verbose if multiple columns are concerned, where it would make sense to have a `dropna` function on the `DataFrame`.

**Comment by Gmousse** [comment]
Hi @episodeyang,

Good idea. I will try it and I will publish a proposal here.

**Comment by Gmousse** [comment]
Hi @episodeyang,
It's done, we have two new methods `.dropMissingValues` and `.fillMissingValues`.

It's on develop and will be released in the 1.4.0 soon.

**Comment by episodeyang** [comment]
Thanks, this is really awesome!

**Comment by Gmousse** [comment]
Released in 1.4.0
